src/policy_evaluator/policy_evaluator_builder.rs

```diff
@@ -126,20 +132,20 @@ impl PolicyEvaluatorBuilder {
     }
 
     /// Ensure the configuration provided to the build is correct
-    fn validate_user_input(&self) -> Result<()> {
+    fn validate_user_input(&self) -> anyhow::Result<()> {
```

Comment:
Why are we still using anyhow here? Is to avoid rewriting all the errors? I'm aware that this errors are internal to the library. But I would like to know if there is some other reason to keep the anyhow reference here. Could not we replace this anyhow error by a custom error where it has different error messages? Maybe the `InvalidUserInput` which is the error that we map these one to. 

I do not consider this suggestion a blocker to merge this PR.

Reply:
indeed, it's because they are internal. It maybe be better to provide another class of errors and do an `#[error("transparent")]`. Will have a look into it.

Reply:
done in https://github.com/kubewarden/policy-evaluator/pull/432/commits/3dac38dacd18f0b45121c3bb820954ac5de60855.